tests/fstrings_test.py

```diff
@@ -30,6 +30,7 @@
         '"{}{}".format(a)', '"{a}{b}".format(a=a)',
         # invald \N escape
         r'"{} \\N{snowman}".format(a)',
+        r'"{} \\N{snowman}".format(a, snowman)',
```

Comment:
can we show the successful case as well, where it gets turned into an f-string but slightly looks like a \N escape?